addons/actions.md

```diff
@@ -168,6 +168,111 @@ end
+- `getNextBankHoliday(<datetime>, <file>)`: returns the name of the next bank holiday after the day defined by the Joda DateTime `<datetime>` defined in `<file>`.
+- `isBankHoliday`: returns `true` if today is a bank holiday (see below); returns `false` otherwise.
+- `isBankHoliday(<offset>)`: returns `true` if the day `<offset>` days from today is a bank holiday; returns `false` otherwise.
+- `isBankHoliday(<datetime>)`: returns `true` if the day defined by the Joda DateTime `<datetime>` is a bank holiday; returns `false` otherwise.
+- `isBankHoliday(<offset>, <file>)`: returns `true` if the day `<offset>` days from today is a day defined in `<file>`; use 0 for `<offset>` for today; returns `false` otherwise.
+- `isBankHoliday(<datetime>, <file>)`: returns `true` if the day defined by the Joda DateTime `<datetime>` is in `<file>`; returns `false` otherwise.
+- `isInDayset("<set>")`: returns `true` if today is in the custom dayset `<set>` (see below), for example `isInDayset("school")`; returns `false` otherwise.
+- `isInDayset("<set>", <offset>)`: returns `true` if the day `<offset>` days from today is in dayset `<set>`; returns `false` otherwise.
+- `isInDayset("<set>", <datetime>)`: returns `true` if the day defined by the passed in Joda DateTime `<datetime>` os in dayset `<set>`; returns `false` otherwise.
+- `isWeekend`: returns `true` if today is a weekend, `false` otherwise.
+- `isWeekend(<offset>)`: returns true if the day `<offset>` days from today is a weekend. Use a negative value to check days in the past.
+- `isWeekend(<datetime>)`: returns `true` if the day defined by the passed in Joda DateTime is a weekend, `false` otherwise.
+
+#### Scripted Automation
+
+One must import the Ephemeris Action and then call the above functions using `Ephemeris.<function>`, for example `Ephemeris.getNextBankHoliday()`.
+
+### Configuration
+
+#### PaperUI
+
+In PaperUI one has the ability to configure which days of the week are considered weekends and set your country, region, and city.
+If no country is provided the service uses your system default locale settings.
+Browse to Configuration and System and scroll down to the Ephemeris section.
+Setting the country, region and optionally city will cause Ephemeris to use the list of bank holidays defined for that location by [Jollyday](https://github.com/svendiedrichsen/jollyday).
+You can find the XML file for your country [here](https://github.com/svendiedrichsen/jollyday/tree/master/src/main/resources/holidays) and the localized mapping files that map between the holiday name in the XML and your preferred language are located [here](https://github.com/svendiedrichsen/jollyday/tree/master/src/main/resources/descriptions).
+
+When filling in the country, region and city, if there isn’t a drop down list for one after choosing the one(s) higher up (e.g. there is no drop down list for city after choosing country and region), leave that field blank.
```

Comment:
Maybe consider using more sentences here.
I find it very hard to read this multiple nested one.

Reply:
This hopefully is addressed in the upcoming commit.